<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>1501</width>
     <height>1001</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="startPage">
    <property name="styleSheet">
     <string notr="true">background-color : #2E1114;
color : #ADADAD;</string>
    </property>
    <widget class="QLabel" name="gameTitleLabel1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>1461</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>18</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Casino Simulator</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="nameTitleLabel1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>1461</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>10</pointsize>
       <italic>true</italic>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>By: Jacoby King</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="toPage2Button1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>820</y>
       <width>1161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #501B1D;</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="gameDescriptionLabel1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>1461</width>
       <height>381</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Welcome to my casino simulator program. This was built as part of my final project
for CIS 164, advanced C++. This program starts the user off with $1,000 to gamble
in two different mini games, slots and horse racing. As the user either gains or loses
cash ,the program will update their standing on the game selection screen as well as
the results of the previous game played. If the player runs out of money after one of
the mini games, they will be taken straight to the end screen which will display all of
your results including profit or loss. Thank you for checking out my game I hope you
enjoy.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gameSelectionPage">
    <property name="styleSheet">
     <string notr="true">background-color : #2E1114;
color : #ADADAD</string>
    </property>
    <widget class="QLabel" name="gameSelectionLabel2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>1461</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>18</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Select a Game or Tap Out</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="toHorseRacingButton2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>580</y>
       <width>1151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #501B1D;</string>
     </property>
     <property name="text">
      <string>Horse Racing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toSlotsButton2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>700</y>
       <width>1151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #501B1D;</string>
     </property>
     <property name="text">
      <string>Slots</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quitButton2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>820</y>
       <width>1151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #501B1D;</string>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentStandingLabel2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>1461</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>10</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Current Standing: $</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gameResultLabel2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>1461</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>10</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horseRacingPage">
    <property name="styleSheet">
     <string notr="true">background-color : #2E1114;
color : #ADADAD</string>
    </property>
    <widget class="QGraphicsView" name="horseRaceView3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>21</y>
       <width>1461</width>
       <height>661</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #a8a9ad;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="raceButton3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>820</y>
       <width>1161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #501B1D;</string>
     </property>
     <property name="text">
      <string>Race</string>
     </property>
    </widget>
    <widget class="QComboBox" name="horseSelectionBox3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>720</y>
       <width>451</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #501B1D;</string>
     </property>
     <item>
      <property name="text">
       <string>Black</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>White</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Blue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yellow</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="betSelectionBox3">
     <property name="geometry">
      <rect>
       <x>961</x>
       <y>720</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #501B1D;</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="betLabel3">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>720</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Bet:  $</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="slotMachinePage">
    <property name="styleSheet">
     <string notr="true">background-color : #2E1114;
color : #ADADAD</string>
    </property>
    <widget class="QGraphicsView" name="slotMachineView4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>1461</width>
       <height>661</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #a8a9ad;</string>
     </property>
    </widget>
    <widget class="QLabel" name="betLabel4">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>720</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Bet:  $</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="betSelectionBox4">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>720</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #501B1D;</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="spinButton4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>820</y>
       <width>1161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #501B1D;</string>
     </property>
     <property name="text">
      <string>Spin</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="endPage">
    <property name="styleSheet">
     <string notr="true">background-color : #2E1114;
color : #ADADAD</string>
    </property>
    <widget class="QLabel" name="endLabel5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>1461</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>18</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Congrats</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="finalCashLabel5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>1461</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>10</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Your Final Cash Amount:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="endButton5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>820</y>
       <width>1161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : #501B1D;</string>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
    <widget class="QLabel" name="finalCashScoreLabel5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>1461</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>$</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="finalProfitLabel5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>1461</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>10</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Your Final Profit Amount:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="finalProfitScoreLabel5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>1461</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>$</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
